### DISCUSSION TITLE
Consider documenting a release process

### DISCUSSION THREAD

**Discussion by wemeetagain** [libp2p/js-libp2p#2149] [General]
In lodestar, having a [documented release process](https://github.com/ChainSafe/lodestar/blob/unstable/RELEASE.md) has helped our team and downsteam consumers have clarity on how new releases get published, how bugfixes get backported, and which published versions are officially supported.

I don't recommend using the lodestar process verbatim, but I think its worth thinking about what kind of process we want.

**Comment by maschad** [comment]
We should  also update the readme to let users know that they can install the latest master via `libp2p@next` or they can install via commits if they want a [specific commit via npm](https://www.npmjs.com/package/libp2p?activeTab=versions)
